p2p.go

```diff
@@ -100,6 +98,7 @@ var (
 		EnableRateLimit:          false,
 		RateLimit:                DefaultRatelimitConfig,
 		PrivateNetworkPSK:        "",
+		DHTProtocolName:          "/iotex",
```

Comment:
1. call it `ProtocolName`? now it used DHT, in theory can be other protocol
2. move to line 48